subsys/ipc/rpmsg_service/rpmsg_backend.c

```diff
@@ -131,6 +131,8 @@ static void virtio_notify(struct virtqueue *vq)
 	uint32_t current_core = sse_200_platform_get_cpu_id();
 
 	status = ipm_send(ipm_handle, 0, current_core ? 0 : 1, 0, 1);
+#elif defined(CONFIG_IPM_STM32_HSEM)
```

Comment:
Could be useful to explain somewhere ( in a comment or in commit message)that the HSEM mailbox is bi-directional and so that the same channel ID can be used in both directions (i.e. for RX and TX virtqueue).

Reply:
That's a good idea. HSEM mailbox doesn't support the mailbox channel model, only can notify to each other. no id select and data transfer.